src/mlpack/methods/ann/activation_functions/gelu_function.hpp

```diff
@@ -84,9 +84,13 @@ class GELUFunction
         arma::pow(1 / arma::cosh(0.0356774 * arma::pow(y, 3) +
         0.797885 * y), 2) + 0.5;
   }
+
+  //! Locally-stored layer name.
+  static constexpr const char* name = "gelu";
```

Comment:
I guess it would be more clear if we would add the same function as for layers here as well: `static const std::string& Name() { return ""gelu}`. Let me know what you think.

Reply:
Right, this is better. Actually I tried to have `static std::string name = "gelu"`, but that was not allowed so I did the following. Thanks for the suggestion.